lib/bundle/dsl.rb

```diff
@@ -74,14 +74,15 @@ def whalebrew(name)
       @entries << Entry.new(:whalebrew, name)
     end
 
-    def tap(name, clone_target = nil)
+    def tap(name, clone_target = nil, options = {})
```

Comment:
```suggestion
    def tap(name, clone_target = nil, force_auto_update = nil)
```

or similar?

Reply:
I could do that if you insist, but I think leaving it as `options = {}` is more consistent with the rest of this file. For example, `mas` also only accepts one possible option right now, `id`, and it also takes an `options` dictionary. But they are equivalent, so whichever you prefer!